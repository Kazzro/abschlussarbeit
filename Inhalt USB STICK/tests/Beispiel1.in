// Auftrag IHK1
900 6
100, 200, 1, Auftrag A
297, 420, 2, Auftrag B