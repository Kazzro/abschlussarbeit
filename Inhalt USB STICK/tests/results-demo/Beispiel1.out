// Auftrag IHK1
Benötigte Länge: 29.7 cm
Benutzte Flaeche: 54.15 %

Positionierung der Kundenaufträge:
    0 0 100 200 - 1 - Auftrag A
    100 0 420 297 - 2 - Auftrag B

Verbleibende Andockpunkte:
    0 200
    520 0
    100 297
